src/CMakeLists.txt

```diff
@@ -86,3 +90,38 @@ endif()
 add_subdirectory(logger)
 add_subdirectory(shaders)
 add_subdirectory(include)
+
+if(KOMPUTE_OPT_INSTALL)
+    if (KOMPUTE_OPT_USE_BUILT_IN_FMT)
+        # We can't export fmt::fmt because it's alias target, so we unwrap the alias and install it.
+        get_target_property(fmt_aliased_target fmt::fmt ALIASED_TARGET)
+        install(TARGETS ${fmt_aliased_target}
+            EXPORT komputeTargets)
+    endif ()
+
+    if(KOMPUTE_OPT_USE_BUILT_IN_VULKAN_HEADER)
+        # We can't export Vulkan::Headers because it's alias target, so we unwrap the alias and install it.
+        get_target_property(vk_header_target Vulkan::Headers ALIASED_TARGET)
+
+        # Before installation, reset the INTERFACE_INCLUDE_DIRECTORIES to a generator expression
+        # Otherwise INTERFACE_INCLUDE_DIRECTORIES will point to an absolute path even if the software package
+        # is copied to another computer. If we don't do this, cmake will complains about it in the configuration step.
+        # Get the build interface of include dir
+        get_target_property(vk_header_include_dir ${vk_header_target} INTERFACE_INCLUDE_DIRECTORIES)
+        #message(STATUS "vk_header_include_dir = ${vk_header_include_dir}")
+        # Reset INTERFACE_INCLUDE_DIRECTORIES to a generator expression.
+        set_target_properties(${vk_header_target}
+            PROPERTIES
+            INTERFACE_INCLUDE_DIRECTORIES "$<INSTALL_INTERFACE:include>;$<BUILD_INTERFACE:${vk_header_include_dir}>")
+        #message(STATUS "vk_header_include_dir = ${vk_header_include_di r}")
+        install(TARGETS ${vk_header_target}
+            EXPORT komputeTargets)
+    endif ()
+
+    if(KOMPUTE_OPT_BUILD_PYTHON)
```

Comment:
Wondering if indeed the python components are to be installed, as normally they are internal for the python installation through pip only - however I also don't see harm on doing this

Reply:
Python components are not to be installed, python libs are found by find_package from external directories, and I don't think we need to install them. It,seems,that we,don't use builtin python, so we don't need to install it.

The install call doesn't install anything about pybind11 either, it just export this empty target into komputeTargets to satisfy the requirement of cmake.

Reply:
Makes sense